app/project/page.tsx

```diff
@@ -34,22 +56,62 @@ const Page = () => {
       </div>
       <div className='relative'>
         <Image src={background} alt={'background'} />
```

Comment:
using image as background might cause layout issue in the future

Reply:
the whole picture is the background, including the pencil, paper and wordings. How may i change this
![project](https://github.com/21ftt1750/tododo/assets/99001373/c9a4d04c-60a3-4081-bd31-872573ca7944)

Reply:
oops this is just meant as a heads up. can merge now.